src/cli.rs

```diff
@@ -131,7 +131,7 @@ pub struct RootOpts {
     pub quiet: u8,
 
     /// Set the logging format
-    #[arg(long, default_value = "text", env = "VECTOR_LOG_FORMAT", value_parser(["text", "json"]))]
+    #[arg(long, default_value = "text", env = "VECTOR_LOG_FORMAT")]
     pub log_format: LogFormat,
```

Comment:
Did you try out removing the `impl FromStr` for these two enums?

Reply:
Dropped the `impl` in favor of `derive(clap::ValueEnum)`